docs/get-prod-db-backup.md

```diff
@@ -0,0 +1,54 @@
+# Getting a local copy of the prod DB
+
+This is a fairly manual process with a tiny bit of rake help at the end. 
+Hopefully we'll automate daily backups. In the meantime, there's this. 
+
+## Create a backup (currently manual)
+
+SSH into bastion with 
+
+`ssh ccs.production`
```

Comment:
Nitpick: this is configuration specific, and other devs might not have the same nickname for the bastion. Is it worth giving some indication that this is a generic "whatever you call your ccs production host"?

Reply:
I really need this on `develop` now as I'm doing a data quality migration. @CristinaRO can I agree to come back to it?